### DISCUSSION TITLE
Can cyclone_dx be parallel?

### DISCUSSION THREAD

**Discussion by jgsuess** [oracle/macaron#346] [Q&A]
It seems that `cyclonedx-gradle` runs late and blocks the build for a while. Is there a reason it has to run at this time, or could it run in parallel earlier?

> 2023-06-28 04:34:27,946 [INFO] Running cyclonedx-gradle version 1.7.4 dependency analyzer on /home/macaron/output/output/git_repos/github_com/micronaut-projects/micronaut-core

**Comment by behnazh-w** [comment]
SBOM generation is used to detect dependencies and has to run after analyzing the main target. You can skip the dependency analysis by passing `--skip-deps`.
